src/_includes/meta_data.html

```diff
@@ -0,0 +1,69 @@
+<!------Start Title Meta------->
```

Comment:
Regarding these comments:
- If they're needed (not sure they are) perhaps we should have multiple includes and turn the comments into the file names, e.g. `meta_titles.html`, `meta_description.html`, `meta_share_images.html`
- Some of the comments are not considered valid XML. The character sequence '--' shouldn't be used inside a comment. This seems to upset some browsers (see image) _This was news to me but looks funky if anyone inspects our source code_
- If anyone does inspect out source code these comments will appear totally out of context as they rap variable definitions that will not appear (see image). This is very minor but I guess some of those customers will be scrutanising our code. It's also unnecessary extra weight in the download.

<img width="1680" alt="Screenshot 2020-04-27 at 14 36 30" src="https://user-images.githubusercontent.com/353044/80378394-82402280-8894-11ea-86d4-c513b9486f4f.png">

Reply:
Resolved by https://github.com/codurance/site/pull/1709/commits/8abbbe522261704d528ac5ec386aa50bdf1d36cf